ara/ui/templates/partials/playbook_results.html

```diff
@@ -149,6 +149,11 @@ <h4>
                     {% else %}
                       {{ result.task.name }}
                     {% endif %}
+                    {% if result.task.tags %}
+                    <span class="badge badge-pill badge-warning" class="btn btn-secondary" data-toggle="tooltip"  data-placement="right" data-html="true" title="{% include 'partials/tags_list.html' with tags=result.task.tags %}">
```

Comment:
I'm OK with this approach but I would tend to use another color here because it overlaps with what we use to highlight the changed status:
![Screenshot from 2021-06-26 21-05-09](https://user-images.githubusercontent.com/1291204/123529836-5d2a8800-d6c2-11eb-9fbb-e45467315056.png)

Looking at the [docs](https://getbootstrap.com/docs/4.0/components/badge/#contextual-variations), maybe ``badge-secondary`` instead of ``badge-warning`` ?